core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -78,8 +78,10 @@
 
     private static final int DEFAULT_NUM_BOSSES = 1;
     private static final int DEFAULT_NUM_WORKERS;
-    private static final int DEFAULT_MAX_PENDING_REQUESTS = 8;
-    private static final int DEFAULT_MAX_CONNECTIONS = 65536;
+
+    // Use Integer.MAX_VALUE not to limit open connections by default.
```

Comment:
Instead of always including the handler even when not limiting, is it possible to default to, e.g. -1 and then just not add the handler to the pipeline when not limiting?